gdk4-macos/src/auto/macos_surface.rs

```diff
@@ -20,14 +20,6 @@ glib::wrapper! {
 }
 
 impl MacosSurface {
-    //#[cfg(feature = "v4_8")]
-    //#[cfg_attr(docsrs, doc(cfg(feature = "v4_8")))]
-    //#[doc(alias = "gdk_macos_surface_get_native_window")]
-    //#[doc(alias = "get_native_window")]
-    //pub fn native_window(&self) -> /*Unimplemented*/Option<Basic: Pointer> {
-    //    unsafe { TODO: call ffi:gdk_macos_surface_get_native_window() }
-    //}
-
     //pub fn native(&self) -> /*Unimplemented*/Basic: Pointer {
```

Comment:
This property has to be ignored as well, and you can use it as a fallback for pre v4.8

Reply:
basically getting the property as a pointer and doing the same as you do when calling gdk_macos_surface_get_native_window

Reply:
Fixed